app/gui2/src/composables/events.ts

```diff
@@ -288,71 +288,81 @@ export const enum PointerButtonMask {
  */
 export function usePointer(
   handler: (pos: EventPosition, event: PointerEvent, eventType: PointerEventType) => void | boolean,
-  requiredButtonMask: number = PointerButtonMask.Main,
-  predicate?: (e: PointerEvent) => boolean,
+  options: {
+    requiredButtonMask?: number
+    pointerCapturedBy?: 'target' | 'currentTarget' | 'none'
+    predicate?: (e: PointerEvent) => boolean
+  } = {},
 ) {
-  const trackedPointer: Ref<number | null> = ref(null)
-  let trackedElement: (Element & GlobalEventHandlers) | null = null
-  let initialGrabPos: Vec2 | null = null
-  let lastPos: Vec2 | null = null
+  const requiredButtonMask = options.requiredButtonMask ?? PointerButtonMask.Main
```

Comment:
The defaults require documentation, as it is not clear now that non-main button events will be ignored by default. Let's make the `options` object into an interface and document its fields.